src/routes/safe/store/reducer/newTransactions.ts

```diff
@@ -0,0 +1,63 @@
+import { handleActions } from 'redux-actions'
+
+import { Transaction } from '../models/types/transactions'
+import { LOAD_MORE_TRANSACTIONS, SET_NEXT_PAGE, SET_PREVIOUS_PAGE } from '../actions/transactionsNew/pagination'
+
+export const TRANSACTIONS = 'transactionsNew'
+
+export interface NewTransactionsState {
+  offset: number
+  limit: number
+  totalTransactionsAmount: number
```

Comment:
are these numbers unique for all safes? honestly I think it should be stored in component's state

Reply:
`totalTransactionsAmount ` yes, it's the amount of txs for that safe

Reply:
Yes, but why do we need to store that at the redux store?